pkg/operator/controllers/allocation_api_server_test.go

```diff
@@ -23,11 +26,20 @@ var _ = Describe("allocation API service input validation tests", func() {
 		sessionID1     string = "d5f075a4-517b-4bf4-8123-dfa0021aa169"
 		gsName         string = "testgs"
 	)
+	//Block of code to make port grabbing dynamic
```

Comment:
we don't need to do this for unit tests, we can have port 5000 in a const and use it.

Reply:
I'll run some more e2e tests and verify, but I believe if the service listens on (for example) 5005 and the e2e tests are set to a const of 5000, they fail. Will double check and push the const if that is incorrect.

Reply:
correct, it will fail. But why would we change the port as part of running the end to end tests? We should change it only on production deployments. 
For local e2e tests running, we can have some documentation on how to change the port.

Reply:
Pushed new changes with port set back to const of 5000